js/views/modals/Settings/Advanced.js

```diff
@@ -47,6 +48,35 @@ export default class extends baseVw {
     return super.getFormData(subset);
   }
 
+  clickPurge() {
+    this.purgeCache();
+  }
+
+  /**
+   * Call to the server to remove cached files that are being shared on IPFS.
+   * This call should not be aborted when the view is removed, it's critical the user is informed if
+   * the call fails, even if they have navigated away from the view.
+   */
+  purgeCache() {
+    this.getCachedEl('.js-purge').addClass('processing');
+    this.getCachedEl('.js-purgeComplete').addClass('hide');
+
+    this.purge = $.post(app.getServerUrl('ob/purgecache'))
+      .always(() => {
+        this.getCachedEl('.js-purge').removeClass('processing');
+      })
+      .fail((xhr) => {
+        const failReason = xhr.responseJSON && xhr.responseJSON.reason || '';
+        openSimpleMessage(
+          app.polyglot.t('settings.advancedTab.server.purgeError'),
+          failReason);
+      })
+      .done(() => {
+        this.getCachedEl('.js-purgeComplete').removeClass('hide');
+      });
+  }
+
+
```

Comment:
Don't forget the super.render().

At some point, I think I'll try and write a test that would check all views and ensure that if they are using getCachedEl, that they are calling super.render(). It's really easy to forget.